src/player.cpp

```diff
@@ -451,6 +451,33 @@ void player::reset_stats()
     Character::reset_stats();
 }
 
+void player::reset_scent()
```

Comment:
That `reset_scent` may cause conflicts with the same `reset_scent` in the mutant dog PR, unless you're basing it on the same commits.
Those will probably be easy to solve conflicts, but it's a good idea to keep PRs separate when possible, or basing one on the other if you're sure enough that the earlier one will get merged.